docs/UserGuide.md

```diff
@@ -437,7 +437,7 @@ except after search and view commands. <br>
   and will no longer be displayed at the top.
   * e.g After a sort command, flagged applicants are not displayed at the top of the list and will be displayed according to the order 
   specified in the command. 
-  * e.g After a search command, applicants will be displayed in order of which applicants most satisfy the search conditions.
+  * e.g After a search command, applicants will be displayed in descending order of number of matches with the search conditions.
```

Comment:
I think maybe an example could support this statement? If I were viewing this from the POV of an outsider, I think I might not fully understand this statement. Can check with @kaiyichen as I suggested her to add an example as well.

Reply:
Asked users to refer to search command for more information as example is already there.